examples/inference/api_server_openai/query_openai_sdk.py

```diff
@@ -41,20 +41,49 @@
 
 args = parser.parse_args()
 
-# List all models.
-models = openai.Model.list()
-print(models)
-
-# Note: not all arguments are currently supported and will be ignored by the backend.
-chat_completion = openai.ChatCompletion.create(
-    model=args.model_name,
-    messages=[
-        {"role": "assistant", "content": "You are a helpful assistant."},
-        {"role": "user", "content": "Tell me a long story with many words."},
-    ],
-    stream=args.streaming_response,
-    max_tokens=args.max_new_tokens,
-    temperature=args.temperature,
-    top_p=args.top_p,
-)
-print(chat_completion)
+version = openai.__version__
+version_content = version.split(".")
+
+if version_content[0] == "0":
+    # List all models.
+    models = openai.Model.list()
+    print(models)
+
+    # Note: not all arguments are currently supported and will be ignored by the backend.
+    chat_completion = openai.ChatCompletion.create(
+        model=args.model_name,
+        messages=[
+            {"role": "assistant", "content": "You are a helpful assistant."},
+            {"role": "user", "content": "Tell me a long story with many words."},
+        ],
+        stream=args.streaming_response,
+        max_tokens=args.max_new_tokens,
+        temperature=args.temperature,
+        top_p=args.top_p,
+    )
+    print(chat_completion)
+elif version_content[0] == "1" and version_content[1] <= "9":
+    from openai import OpenAI
+
+    client = OpenAI()
+    # # List all models.
+    models = client.models.list()
```

Comment:
can we just limit openAI sdk version to >= 1.0 and always use this method? We don't need to limit the version to < 1.9 if possible.

Reply:
Do we need support for version <1.0？